### PR TITLE
Fixes to the Quicklab install of ArgoCD.

### PR DESCRIPTION
**Pull request by martinpovolny**
A few issues I hit into when trying the HOWTO.

Ping @tumido

### CODE REVIEW

File: docs/downstream/quicklab.md [hunk 1]
```diff
@@ -25,11 +25,18 @@
...
+   oc login upi-0.tcoufaltest.lab.upshift.rdu2.redhat.com:6443
+   ```
+
 ## Install Argo CD on your cluster
 
-1. kube:admin is not supported in user api, therefore user has to create additional user. Simplest way is to deploy an Oauth via Htpasswd:
+1. kube:admin is not supported in user api, therefore you have to create additional user. Simplest way is to deploy an Oauth via Htpasswd:
 
-2. Create a htpasswd config file and deploy it to OpenShift:
+1. Create a htpasswd config file and deploy it to OpenShift:
```
Comment (tumido) [inline docs/downstream/quicklab.md]: Please undo all those numbering changes

**Review by tumido — COMMENTED** [review]
Just a nitpick. There's no reason to undo the ordered lists numbering... Otherwise LGTM.

File: docs/downstream/quicklab.md (hunk 1 above)
Comment (martinpovolny) [reply docs/downstream/quicklab.md]: Why that? The implicit numbering does now work for you?

**Review by tumido — APPROVED** [review]

**Review by tumido — CHANGES_REQUESTED** [review]
Precommit fails on this change set, please fix the trailing whitespace:
```
Trim Trailing Whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing docs/downstream/quicklab.md
```

**Review by tumido — APPROVED** [review]
LGTM :+1: 

Note: found precommit violations are located in files not modified in this PR.

**Pull request merged**